otx/algorithms/anomaly/adapters/anomalib/exportable_code/base.py

```diff
@@ -23,10 +23,7 @@ def parameters(cls):
         parameters["resize_type"].update_default_value("standard")
         parameters.update(
             {
-                "image_threshold": NumericalValue(description="Image-level Threshold value to locate anomaly"),
```

Comment:
Are these changes backward compatible? Sometimes OTX updates break loading legacy projects when upgrading Geti version

Reply:
Not really. Do Geti projects rely on exportable code?

Reply:
No, they don't, but I mean compatibility in general. Geti saves ModelEntities in it's database as a part of a project. OTX should be compatible with ModelEntities from previous Geti versions. We already had an issue with missing extra configs for tiling, so this question is just a precaution. If you don't see any problems with Geti upgrade scenario, that's fine.

Reply:
I have a feeling that it might affect Geti upgrade. But maybe we will have to fix it only once. The advantage of having metadata as a json is that we can be flexible later on if keys are added or removed. Feel free to ping me if this breaks during Geti project upgrade.